runtime/commonTest/src/kotlinx/serialization/features/ObjectSerialization.kt

```diff
@@ -48,16 +46,4 @@ class ObjectSerializationTest : JsonTestBase() {
             json
         )
     }
-
-    @Test
-    fun correctDescriptor() {
```

Comment:
why you've deleted this test?

Reply:
I've added a separate test to SerialDescriptorSpecificationTest